### ISSUE TITLE
False negatives with `operator-no-unspaced` rule for `unicode-range` values in `@font-face`

### ISSUE THREAD

**Issue by davidding** [kristerkari/stylelint-scss#261]
Using the example from https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range#Examples:

```
@font-face {
  font-family: 'Ampersand';
  src: local('Times New Roman');
  unicode-range: U+26;
}
```

yields unexpected errors with `stylelint-scss`, `v3.1.3`:

```
  4:19  ✖  Expected single space before "+"     scss/operator-no-unspaced
  4:19  ✖  Expected single space after "+"      scss/operator-no-unspaced
```

([W3C `unicode-range` reference](https://www.w3.org/TR/2018/CR-css-fonts-3-20180626/#unicode-range-desc))

**Comment by kristerkari** [comment]
Thanks!

**Comment by kristerkari** [comment]
PR with a fix:
https://github.com/kristerkari/stylelint-scss/pull/262
